tasks/collect-collectors-data/README.md

```diff
@@ -0,0 +1,18 @@
+# collect-collectors-data
+
+Tekton task to collect the information related to collectors.
+
+The purpose of this task is to collect all the data and supply it to other tasks in the pipeline by creating
+json files for the different resources.
+
+A task result is returned for each resource with the relative path to the stored JSON for it in the workspace.
```

Comment:
WDYT about adding a note like: "This task is similar to the `collect-data` task, but it only collects a subset of the data."

I think it would be nice, but others might feel differently.

Reply:
Both addressed.